homeassistant/components/google_generative_ai_conversation/const.py

```diff
@@ -23,14 +24,15 @@
 {%- endfor %}
 """
 
+CONF_RECOMMENDED = "recommended"
+RECOMMENDED_LABEL = "Recommended (Gemini 1.5 Flash)"
 CONF_CHAT_MODEL = "chat_model"
-DEFAULT_CHAT_MODEL = "models/gemini-pro"
+RECOMMENDED_CHAT_MODEL = "models/gemini-1.5-flash-latest"
 CONF_TEMPERATURE = "temperature"
-DEFAULT_TEMPERATURE = 0.9
+RECOMMENDED_TEMPERATURE = 0.9
 CONF_TOP_P = "top_p"
-DEFAULT_TOP_P = 1.0
+RECOMMENDED_TOP_P = 1.0
 CONF_TOP_K = "top_k"
-DEFAULT_TOP_K = 1
+RECOMMENDED_TOP_K = 1
```

Comment:
Similarly change this to 64. https://aistudio.google.com/ hides the Top K option when you select 1.5 pro or flash and the default of 1.0 is None. Not sure how I had default 1 here. Even the original models/chat-bison-001 had 40 here. Maybe that was the default in https://aistudio.google.com/ since if I remember right I used the default values from there.